src/browser/api/notifications/subscriptions.ts

```diff
@@ -305,9 +305,13 @@ function requestNoteCreation (noteData: any, parent: any) {
     } else {
         const {ack} = noteData;
 
-        ++askedFor;
-        notesToBeCreated.push(name);
-        invokeCreateAck(ack);
+        try {
+            invokeCreateAck(ack);
```

Comment:
Why is the order changed here?

Reply:
The order changed, because `invokeCreateAck` can throw an error, and if it does, then we **don't want** to proceed with `++askedFor;` and `notesToBeCreated.push(name);`